enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -0,0 +1,415 @@
+modifications to the parent policy specification while the child
+policy is in enforcing mode.
+
+The values of the annotation can take either of these forms:
++ The value of `any` indicates that changes to the parent policy are
+immediately passed through to the enforcement of the child policy. As
+long as this annotation has the value of `any` remediation is enabled
+on the child policy and changes to the parent policy are immediately
+enforced to the cluster.
++ An integer `generation` value indicates that the child policy will
+continue enforcing as long as the child policy `metadata/generation`
+value is equal to the specified value. In this way the user of this
+feature has control over whether changes to the parent policy happen
+immediately or require additional intervention by the user. At the
+expense of this additional complexity the behavior closes race
+conditions on a user/controller enabling/disabling enforcement and
+near-concurrent changes to the parent policy specification.
+
+The overall pattern follows a similar one in OLM operator updates
+where a manual installPlanApproval is explicitly enabled by the user
+when an update is desired.
+
+The implementation is relatively simple and defers the “selection”
+flexibility to the user/orchestrator/controller
+
+To improve user visibility the dashboard view of policies should be
+enhanced to show the current enforcement state of the policy. This
+would be reflected in the Clusters tab of the Policies/_policyName_
+page and the specific child policy view
+Policies/_policyName_/_cluster-policy-name_
+
+### User Stories
+
+Detail the things that people will be able to do if this is
+implemented.  Include as much detail as possible so that people can
+understand the "how" of the system. The goal here is to make this feel
+real for users without getting bogged down.
+
+#### Story 1
+I have a custom controller/operator which is responsible for applying
+a configuration change to my fleet of clusters. SLAs require a
+progressive rollout to my network. The rollout proceeds as follows:
+1. I define the desired configuration of my clusters in a
+   Configuration Policy which I bind to a large set of clusters in my
+   network.
+1. The policy is created in inform mode and I can see/verify the scope
+   and impact of the changes.
+1. My controller/operator selects, based on its criteria, a set of
+   clusters which should receive the configuration update.
+1. My controller/orchestrator annotates the child policy for the
+   selected clusters with `versioned-policy-enable: <value>`
+1. The GRC policy controller remediates the policy on only the
+   selected clusters according to existing "enforce" mode rules. The
+   other bound clusters continue evaluating the policy under "inform"
+   mode rules.
+1. My controller recognizes the child policy moving to the compliant
+   state and removes the `versioned-policy-enable` annotation.
+1. The GRC policy controller reverts to existing "inform" mode
```

Comment:
My comment (https://github.com/open-cluster-management-io/enhancements/pull/57/files#r892335110) doesn't account for this. Perhaps a different `remediationAction` on the parent policy of `controlledInformOnCompliant` would suffice to capture this additional desired behavior.